modules/docopt.py

```diff
@@ -1,66 +0,0 @@
-"""
-Usage:
-  RoomMgt
-  RoomMgt create_room <room_type> <room_name> ...
-
```

Comment:
Try working on more functionality because you have only implemented one on your docopt. I think its still a requirement to complete all functionality given so as to successfully complete CP1